tests/unit/test_plugins_cis_audit.py

```diff
@@ -42,19 +42,24 @@ class TestCronCisAudit(TestCase):
     """Test the cis-audit cron job functions."""
 
     cloud_init_logfile = os.path.join(tempfile.gettempdir(), "cloud-init-output.log")
-    bionic_profiles = [
-        "level1_server",
-        "level2_server",
-        "level1_workstation",
-        "level2_workstation",
-    ]
+    bionic_profiles = {
+        "level1_server": "level1_server",
+        "level2_server": "level2_server",
+        "level1_workstation": "level1_workstation",
+        "level2_workstation": "level2_workstation",
+    }
     bionic_audit_folder = "/usr/share/ubuntu-scap-security-guides"
     bionic_audit_result_glob = bionic_audit_folder + "/cis-*-results.xml"
     bionic_audit_bin = ["/usr/sbin/cis-audit"]
 
     focal_audit_folder = "/var/lib/usg/"
     focal_audit_result_glob = focal_audit_folder + "/usg-results-*.*.xml"
-    focal_profiles = ["cis_" + p for p in bionic_profiles]
+    focal_profiles = {
+        "level1_server": "cis_level1_server",
+        "level2_server": "cis_level2_server",
+        "level1_workstation": "cis_level1_workstation",
+        "level2_workstation": "cis_level2_workstation",
+    }
```

Comment:
This kind of repetition (redefining values already present in the source code) should be avoided, as we're reimplementing code from the source and tests become less declarative  - I've left a suggestion about mocking `_get_major_version` instead of `_valid_profiles_for_platform` to avoid this here.